pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,10 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            data = base64.b64decode(api_repr.get('data', b''))
+        except (binascii.Error, TypeError):
+            data = api_repr.get('data', b'')
```

Comment:
@daspecster RE: `decode_base64()`. Hold off on it for now. **IF** we hear another report like this, we can look into making a shared helper.